pegasus/sites.v3/code.org/public/challenge.md

```diff
@@ -1,19 +1,22 @@
 ---
+<%
+  share_image = DCDO.get('soon-hoc', false) ? 'hoc-student-challenge.jpg' : 'celeb-challenge.jpg'
+%>
 title: #HourOfCode Challenge
 theme: responsive
 style_min: true
 social:
   "og:title": "Celebrity Challenge"
   "og:description": "Win a celebrity video chat for your class!"
-  "og:image" : "<%= 'https://' + request.host + '/images/social-media/hoc-student-challenge.png' %>"
+  "og:image" : "<%= 'https://' + request.host + '/images/fit-1920/social-media/' + share_image %>"
```

Comment:
1920 seems really large for a Twitter/OG image. Facebook seems to recommend min 1200 x 630 and Twitter 1024 x 512 pixels. @tanyaparker may know more.

Reply:
If we do keep it 1920, are both the images at least 1920?

Reply:
Yep!  I added this because both images are 2mb+ 😕.  We can go smaller, but I added an upper bound to match the existing `og:image:width` size.

Reply:
Hopefully production will serve them smaller than 2mb+ when we use `fit-1920` and serve as JPGs?